File: cgi-bin/ruby/processdata.rb
#!/usr/bin/ruby -w
require 'cgi';
require 'bigdecimal';
cgi = CGI.new;

print "X-Complexity: Easy\n";
print "Content-type: text/html \n\n";
print "<!doctype html>";
print  "<html><head><title>Hello Ruby based world </title></head>" ;
h = cgi.params;
m1 = Integer(h['magicnumber'][0]);


while m1 > 0 do
 print "<h1>Hello " +  h['username'][0] + "with a password of " + h['password'][0] + "!</h1>";
 m1 = m1 - 1;
end

print "</body></html>";

File: cgi-bin/py/processdata.py
#!/usr/bin/python

from random import randrange
from datetime import datetime
from pytz import timezone

import cgi
import cgitb

form = cgi.FieldStorage()

count = form.getvalue('magicnumber')
name = form.getvalue('username')
pw = form.getvalue('password')


print "X-Complexity: Easy"
print "Content-type: text/html"
print ""
print "<html>"
print "<head><title>Hello Python Based CGI World!</title></head>"
count = int(count) + 0

while (count > 0):
  count = count - 1
  print "<h1>Hello %s with a password of %s !</h1>" % (name, pw)

print "</body>"
print "</html>"
